src/descriptors/power_system_structs.json

```diff
@@ -706,6 +775,22 @@
         "null_value": "Bus(nothing)",
         "data_type": "Bus"
       },
+      {
+        "name": "model",
+        "comment": "[Z, I, P]",
```

Comment:
I think a comment like this can do more harm than good.  If someone updates the enum, they will almost certainly not update this comment.  Suggest deleting it.  The user can jump-to-definition on the enum.